HIP 128: Energy Subnetwork (#1050)
helium/HIP: XXXX-energy-subnetwork.md

# HIP-XXXX: Energy Subnetwork

- Author(s): [@frahlg](https://github.com/frahlg), [@jthiller](https://github.com/jthiller), [@abhay](https://github.com/abhay), [@T0bias0lss0n](https://github.com/h0bb3), [@leitet](https://github.com/leitet), [@VIKOLO](https://github.com/VIKOLO) 
- Start Date: [Today's Date]
- Category: Economic, Technical
- Original HIP PR: [Leave blank for now]
- Tracking Issue: [Leave blank for now]
- Vote Requirements: HNT Holders

> # Overview

Reviewer: ```suggestion
## Summary
```

* **Stabilize the Power Grid:** Ensure grid stability as the integration of renewable energy sources increases.
* **Empower Homeowners:** Enable homeowners to participate in and benefit from energy markets traditionally dominated by centralized energy companies.

### Market Potential
The global market for ancillary services had a capitalization of over $9 billion in 2023, expected to grow to $18 billion by 2032. In the US, the Power Ancillary Service Market was valued at $6.09 billion in 2022 and is projected to reach $8.79 billion by 2028, with a compound annual growth rate of 6.14%. Today, there are no services available for aggregating a massive amount of small DER’s, globally and with an open and permissionless protocol. Additionally, the production of energy data represents a new market beyond traditional energy sales, providing further compensation opportunities for participants of the ENERGY network.

## Economic Overview

There are a number of real world physical entities that comprise the network and perform the following functional and economic roles:

> | Name                     | Functional Role                                                                                                                                                                                          | Economic Role                                                                                                                                                                                                    |
> |--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
> | DER Host                 | Operates a connected DER through an ENERGY-capable Gateway including compatible IOT Hotspots (Dual Mining).                                                                                                                                                      | Receives ENERGY rewards for sharing data from DER. Stakes ENERGY to qualify DERs for data and VPP service usage and receives ENERGY rewards for participating with controllable DER in the VPP (Energy services). |                                                                                                           |
> | Service Provider         | Promotes and enables data and VPP services to Clients. Aggregates DERs for market optimizations and guarantees technical requirements. Srcful AB is the first service provider at ENERGY network launch. | Receives payments for data and VPP services from Clients, burns HNT into Data Credits and settles data usage with DER Hosts. Stakes ENERGY to receive % of rewards from Energy Emissions contract.               |
> | Client                   | Energy Companies, Grid Operators, or Aggregators that use the ENERGY network. Clients act in accordance with the country's Transmission System Operator.                                           | Pays Service Providers to access data and/or the right to control DERs in the ENERGY network for VPP services.                                                                                                          |
> | Gateway Vendors          | Builds, sells and supports ENERGY-capable gateways. This includes firmware upgrades for existing Helium Hotspots to enable ENERGY functionality (Dual Mining).                                                                                                                                                      | Stakes ENERGY to access an NFT to be accepted to operate on the ENERGY network.                                                                                                                                          |
> | Sensors and Support Host | Provides needed high-quality data (e.g Frequency meters) to support the VPP, and special reward programs.                                                                                                            | Receives ENERGY mining rewards for verified sensor data or validation and support of the ENERGY network.                                                                                                                |
> | Oracles                  | Relay data to L1 emission contract. Calculate incentive points for service providers and sensor hosts.                                                                                                   | Stakes ENERGY. Receive % of rewards from emissions contract                                                                                                                                                      |

Reviewer 2: ```suggestion
| Name                     | Functional Role                                                                                                                                                                                          | Economic Role                                                                                                                                                                                                     |
|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| DER Host                 | Operates a connected DER through an ENERGY-capable Gateway including compatible IOT Hotspots (Dual Mining).                                                                                              | Receives ENERGY rewards for sharing data from DER. Stakes ENERGY to qualify DERs for data and VPP service usage and receives ENERGY rewards for participating with controllable DER in the VPP (Energy services). |                                                                                                           |
| Service Provider         | Promotes and enables data and VPP services to Clients. Aggregates DERs for market optimizations and guarantees technical requirements. Srcful AB is the first service provider at ENERGY network launch. | Receives payments for data and VPP services from Clients, burns HNT into Data Credits and settles data usage with DER Hosts. Stakes ENERGY to receive % of rewards from Energy Emissions contract.                |
| Client                   | Energy Companies, Grid Operators, or Aggregators that use the ENERGY network. Clients act in accordance with the country's Transmission System Operator.                                                 | Pays Service Providers to access data and/or the right to control DERs in the ENERGY network for VPP services.                                                                                                    |
| Gateway Vendors          | Builds, sells and supports ENERGY-capable gateways. This includes firmware upgrades for existing Helium Hotspots to enable ENERGY functionality (Dual Mining).                                           | Stakes ENERGY to access an NFT to be accepted to operate on the ENERGY network.                                                                                                                                   |
| Sensors and Support Host | Provides needed high-quality data (e.g Frequency meters) to support the VPP, and special reward programs.                                                                                                | Receives ENERGY mining rewards for verified sensor data or validation and support of the ENERGY network.                                                                                                          |
| Oracles                  | Relay data to L1 emission contract. Calculate incentive points for service providers and sensor hosts.                                                                                                   | Stakes ENERGY. Receive % of rewards from emissions contract                                                                                                                                                       |
```

> | Name                     | Functional Role                                                                                                                                                                                          | Economic Role                                                                                                                                                                                                    |
> |--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
> | DER Host                 | Operates a connected DER through an ENERGY-capable Gateway including compatible IOT Hotspots (Dual Mining).                                                                                                                                                      | Receives ENERGY rewards for sharing data from DER. Stakes ENERGY to qualify DERs for data and VPP service usage and receives ENERGY rewards for participating with controllable DER in the VPP (Energy services). |                                                                                                           |
> | Service Provider         | Promotes and enables data and VPP services to Clients. Aggregates DERs for market optimizations and guarantees technical requirements. Srcful AB is the first service provider at ENERGY network launch. | Receives payments for data and VPP services from Clients, burns HNT into Data Credits and settles data usage with DER Hosts. Stakes ENERGY to receive % of rewards from Energy Emissions contract.               |
> | Client                   | Energy Companies, Grid Operators, or Aggregators that use the ENERGY network. Clients act in accordance with the country's Transmission System Operator.                                           | Pays Service Providers to access data and/or the right to control DERs in the ENERGY network for VPP services.                                                                                                          |
> | Gateway Vendors          | Builds, sells and supports ENERGY-capable gateways. This includes firmware upgrades for existing Helium Hotspots to enable ENERGY functionality (Dual Mining).                                                                                                                                                      | Stakes ENERGY to access an NFT to be accepted to operate on the ENERGY network.                                                                                                                                          |
> | Sensors and Support Host | Provides needed high-quality data (e.g Frequency meters) to support the VPP, and special reward programs.                                                                                                            | Receives ENERGY mining rewards for verified sensor data or validation and support of the ENERGY network.                                                                                                                |
> | Oracles                  | Relay data to L1 emission contract. Calculate incentive points for service providers and sensor hosts.                                                                                                   | Stakes ENERGY. Receive % of rewards from emissions contract                                                                                                                                                      |

Reviewer: ```suggestion
| Name                     | Functional Role                                                                                                                                                                                          | Economic Role                                                                                                                                                                                                     |
|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| DER Host                 | Operates a connected DER through an ENERGY-capable Gateway including compatible IOT Hotspots (Dual Mining).                                                                                              | Receives ENERGY rewards for sharing data from DER. Stakes ENERGY to qualify DERs for data and VPP service usage and receives ENERGY rewards for participating with controllable DER in the VPP (Energy services). |                                                                                                           |
| Service Provider         | Promotes and enables data and VPP services to Clients. Aggregates DERs for market optimizations and guarantees technical requirements. Srcful AB is the first service provider at ENERGY network launch. | Receives payments for data and VPP services from Clients, burns HNT into Data Credits and settles data usage with DER Hosts. Stakes ENERGY to receive % of rewards from Energy Emissions contract.                |
| Client                   | Energy Companies, Grid Operators, or Aggregators that use the ENERGY network. Clients act in accordance with the country's Transmission System Operator.                                                 | Pays Service Providers to access data and/or the right to control DERs in the ENERGY network for VPP services.                                                                                                    |
| Gateway Vendors          | Builds, sells and supports ENERGY-capable gateways. This includes firmware upgrades for existing Helium Hotspots to enable ENERGY functionality (Dual Mining).                                           | Stakes ENERGY to access an NFT to be accepted to operate on the ENERGY network.                                                                                                                                   |
| Sensors and Support Host | Provides needed high-quality data (e.g Frequency meters) to support the VPP, and special reward programs.                                                                                                | Receives ENERGY mining rewards for verified sensor data or validation and support of the ENERGY network.                                                                                                          |
| Oracles                  | Relay data to L1 emission contract. Calculate incentive points for service providers and sensor hosts.                                                                                                   | Stakes ENERGY. Receive % of rewards from emissions contract                                                                                                                                                       |
```

Reviewer: reformat table

To operate on the network, Service Providers and Gateway Vendors must control a respective NFT. Acquiring this NFT requires Gateway Vendors and Service Providers to:

1. Gateway Vendors must stake 50M ENERGY, and new Service Providers must stake 500M ENERGY.

> 1. Obtain ENERGY governance approval (Srcful AB will be the first Service Provider when the network launches).

Reviewer: ```suggestion
2. Obtain ENERGY governance approval (Srcful AB will be the first Service Provider when the network launches).
```

## Token Allocation
Allocation of ENERGY tokens prioritizes emission to the community of DER hosts for the data and energy services they provide. A max supply of 250B ENERGY tokens is enforced following the mechanics deployed on the IOT and MOBILE subnetworks. 190B tokens are dedicated to emissions mainly as rewards for DER hosts including a one time issuance of 1B tokens to the Operations Fund. 60B tokens are dedicated as emissions for initial funding to investors and founders of the project.

### Emissions Curve

There will be a max supply of 250B ENERGY.

> The proposal is for halvings of ENERGY issuance to occur every 2 years, with the first halvening aligned with the HNT issuance halvening in August 2027. This alignment necessitates an initial period of approximately 3 years from the ENERGY token generation event to the first halvening, followed by regular 2-year halvening intervals thereafter.

Reviewer: ```suggestion
The proposal is for halving of ENERGY issuance to occur every 2 years, with the first halving aligned with the HNT issuance halving in August 2027. This alignment necessitates an initial period of approximately 3 years from the ENERGY token generation event to the first halving, followed by regular 2-year halving intervals thereafter.
```

> We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY network and develop the technical and operational infrastructure required to build the ENERGY subnetwork. The pool will have a lockup of a minimum of 12 months after token emission date. 190B ENERGY is emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts, the baseline rewards for participating in the network and the service rewards for participating in ENERGY services. The baseline rewards will receive 15% of emitted ENERGY and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, remaining tokens are burned. We use net emissions to replenish burned tokens into service rewards using a 30 day average of burned tokens with a cap. The cap will initially be set to 1% of current emissions but will likely be subject to change.

Reviewer: ```suggestion
We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY network and develop the technical and operational infrastructure required to build the ENERGY subnetwork. The pool will have a lockup of a minimum of 12 months after token emission date. 190B ENERGY is emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts, the baseline rewards for participating in the network and the service rewards for participating in ENERGY services. The baseline rewards will receive 15% of emitted ENERGY and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, remaining tokens are burned. We use net emissions to replenish burned tokens into service rewards using a 30-day average of burned tokens with a cap. The cap will initially be set to 1% of current emissions but will likely be subject to change.
```

We propose using one pool for buffering issued baseline rewards:

**Network growth pool** acts as a resource for growing the network using direct incentives to DER hosts. This pool will receive tokens from the baseline DER host rewards inversely proportional to the total ENERGY network DC burn as long as this is less than $100 000 USD. The proportion allocated to the pool is calculated by:

Network&nbsp;Growth&nbsp;Pool&nbsp;Proportion =&nbsp;0.5&nbsp;&minus;&nbsp;min(0.5,&nbsp;0.5&nbsp;*&nbsp;Total&nbsp;ENERGY&nbsp;Network&nbsp;DC&nbsp;burn&nbsp;/&nbsp;1e10)

The motivation for the network growth pool is to stimulate growth of the network in general and to build critical amounts of DERs in areas that will unlock service contracts. DER hosts are rewarded from the pool for participating as a beta tester, for DERs located in hexes whitelisted for services, and to provide current IOT Hotspot hosts one free DER onboarding for a limited time. Other future initiatives with rewards from the network growth pool will be voted on based on suggestions from the community.

For clarity, the community emission schedule is as follows:

> | Year            | ENERGY at the start of the year (M) | ENERGY minted (M) | DER host Baseline Rewards | DER host Service Contract Pool | Sensor hosts | Service Providers | Oracles | Operations Fund | veENERGY stakers |

Reviewer: ```suggestion
| Year            | ENERGY at the start of the year (M) | ENERGY minted (M) | DER host Baseline Rewards | DER host Service Contract Pool | Sensor hosts | Service Providers | Oracles | Operations Fund | veENERGY Stakers |
```

> | Year | ENERGY at the start of the year (M) | ENERGY minted (M) | DER host Baseline Rewards | DER host Service Contract Pool | Sensor hosts | Service Providers | Oracles | Operations Fund | veENERGY stakers |
> |------|-------------------------------------|-------------------|---------------------------|-------------------------------|--------------|-------------------|---------|-----------------|------------------|
> | 1 (Stub + 2025) | 1&nbsp;000 | 47&nbsp;250 | 15 % | 45 % | 5 %  | 20 %  | 4 %  | 5 % | 6 % |
> | 2 (2026)   | 48&nbsp;250                     | 47&nbsp;250   | 15 %                    | 45 %                           | 5 %           | 20 %               | 4 %      | 5 %              | 6 %               |
> | 3 (2027)   | 95&nbsp;500                     | 23&nbsp;625   | 15 %                        | 45 %                           | 5 %           | 20 %               | 4 %      | 5 %              | 6 %               |
> | 4 (2028)   | 119&nbsp;125                    | 23&nbsp;625            | 15 %                        | 45 %                           | 5 %           | 20 %               | 4 %      | 5 %              | 6 %               |
> | 5 (2029)   | 142&nbsp;750                            | 11&nbsp;812.5          | 15 %                        | 45 %                           | 5 %           | 20 %               | 4 %      | 5 %              | 6 %               |
> | 6 (2030)   | 154&nbsp;562.5                          | 11&nbsp;812.5          | 15 %                        | 45 %                           | 5 %           | 20 %               | 4 %      | 5 %              | 6 %               |
> | 7 (2031)   | 166&nbsp;375                            | 5&nbsp;906.25          | 15 %                        | 45 %                           | 5 %           | 20 %               | 4 %      | 5 %              | 6 %               |
> | …    |                                     |                   |                           |                               |              |                   |         |                 |                  |

Reviewer: ```suggestion
| Year            | ENERGY at the start of the year (M) | ENERGY minted (M) | DER host Baseline Rewards | DER host Service Contract Pool | Sensor hosts | Service Providers | Oracles | Operations Fund | veENERGY stakers |
|-----------------|-------------------------------------|-------------------|---------------------------|--------------------------------|--------------|-------------------|---------|-----------------|------------------|
| 1 (Stub + 2025) | 1&nbsp;000                          | 47&nbsp;250       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| 2 (2026)        | 48&nbsp;250                         | 47&nbsp;250       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| 3 (2027)        | 95&nbsp;500                         | 23&nbsp;625       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| 4 (2028)        | 119&nbsp;125                        | 23&nbsp;625       | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| 5 (2029)        | 142&nbsp;750                        | 11&nbsp;812.5     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| 6 (2030)        | 154&nbsp;562.5                      | 11&nbsp;812.5     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| 7 (2031)        | 166&nbsp;375                        | 5&nbsp;906.25     | 15 %                      | 45 %                           | 5 %          | 20 %              | 4 %     | 5 %             | 6 %              |
| …               |                                     |                   |                           |                                |              |                   |         |                 |                  |
```

Reviewer: reformat table

    HETEmissions -->|4%|Oracles
    SensorsSupportHosts --> |Unused tokens|RewardOrBurn

    NetEmissions --> |min#40;avg#40;30 day burn#41;, cap#41;|RewardOrBurn 

    RewardOrBurn -->|All fulfilled service contracts are rewarded according\n to their respective DC burn Remaining tokens are burned|DERHosts

```

### Baseline Rewards
The ENERGY subnetwork proposes an incentive system designed to optimize the participation and contribution of DERs. This approach aims to strengthen and connect more DERs, helping to integrate renewable energy into the grid effectively. The ENERGY subnetwork reward structure draws inspiration from DIMO's successful [Baseline Issuance model](https://docs.dimo.zone/governance/dip2), and rewards DERs for their consistent connectivity and high quality contribution to the network. This structured reward system is crucial for building a reliable data foundation that enables in-depth knowledge of the grid and new service proposals. 

Baseline rewards for an online DER are defined in three buckets based on the following principles:

#### Principle 1: Uptime
Reward DERs that have been part of the network for a long time;

> | Level | Definition                     | Score |
> | ----- | ------------------------------ | ----- |
> | 1     | 3-20 weeks connected           | 100   |
> | 2     | 21-35 weeks connected          | 200   |
> | 3     | 36 or more weeks connected     | 300   |

Reviewer: ```suggestion
| Level | Definition                 | Score |
|-------|----------------------------|-------|
| 1     | 3-20 weeks connected       | 100   |
| 2     | 21-35 weeks connected      | 200   |
| 3     | 36 or more weeks connected | 300   |
```

#### Principle 2: Quality
Incentivize an evenly distributed and continuous data connection; which tells a fuller story of DER behavior and performance. A given time period with data points is subdivided into several levels. If there is data across a whole level the score for that level is used. The score is proportionate to the division of the level. For an hour time period the first division is 2 (data every half hour), the next 2 (data every quarter), the next 3 (data every 5 min), the next 5 (data every min), the next 2 (data every 30 seconds). This gives the following:

> | Level | Definition                 | Score |
> | ----- | -------------------------- | ----- |
> | 1     | Data in each 30 min period | 10    |
> | 2     | Data in each 15 min period | 20    |
> | 3     | Data in each 5 min period  | 60    |
> | 4     | Data in each 1 min period  | 300   |
> | 5     | Data in each 30 sec period | 600   |

Reviewer: ```suggestion
| Level | Definition                 | Score |
|-------|----------------------------|-------|
| 1     | Data in each 30 min period | 10    |
| 2     | Data in each 15 min period | 20    |
| 3     | Data in each 5 min period  | 60    |
| 4     | Data in each 1 min period  | 300   |
| 5     | Data in each 30 sec period | 600   |
```

#### Principle 3: Value
Incentivize DERs who contribute with marketable services such as credible and rapidly controllable resources;
| Level | Definition                                       | Score |
| ----- | ------------------------------------------------ | ----- |
| 1     | Read DER Data                                    | 100   |
| 2     | Read Validated DER Data                          | 300   |
| 3     | Read Validated DER Data + Control Qualified DERs | 600   |

##### Validation and Qualification Mechanism

> Validation and Qualification is offered by the Service Provider to DERs that want to participate in Energy Services. Validation is achieved by the DER Host staking $10 USD worth of ENERGY tokens per kW available power for 45 days with a 10 day un-staking period. During staking the DER is subject to additional Service Provider audits and tests. The DER Host must comply with requirements and provide needed information promptly. Failure to comply with the requirements the staked ENERGY tokens will be forfeited and burned. A validated and controlable DER can be Qualified by the Service Provider. A Qualified DER can be opted in to be available for energy services by the DER Host.

Reviewer 3: ```suggestion
Validation and Qualification is offered by the Service Provider to DERs that want to participate in Energy Services. Validation is achieved by the DER Host staking $10 USD worth of ENERGY tokens per kW available power for 45 days with a 10 day un-staking period. During staking the DER is subject to additional Service Provider audits and tests. The DER Host must comply with requirements and provide needed information promptly. Failure to comply with the requirements the staked ENERGY tokens will be forfeited and burned. A validated and controllable DER can be Qualified by the Service Provider. A Qualified DER can be opted in to be available for energy services by the DER Host.
```

> Validation and Qualification is offered by the Service Provider to DERs that want to participate in Energy Services. Validation is achieved by the DER Host staking $10 USD worth of ENERGY tokens per kW available power for 45 days with a 10 day un-staking period. During staking the DER is subject to additional Service Provider audits and tests. The DER Host must comply with requirements and provide needed information promptly. Failure to comply with the requirements the staked ENERGY tokens will be forfeited and burned. A validated and controlable DER can be Qualified by the Service Provider. A Qualified DER can be opted in to be available for energy services by the DER Host.

Reviewer: ```suggestion
Validation and Qualification is offered by the Service Provider to DERs that want to participate in Energy Services. Validation is achieved by the DER Host staking $10 USD worth of ENERGY tokens per kW available power for 45 days with a 10 day un-staking period. During staking the DER is subject to additional Service Provider audits and tests. The DER Host must comply with requirements and provide needed information promptly. Failure to comply with the requirements the staked ENERGY tokens will be forfeited and burned. A validated and controllable DER can be Qualified by the Service Provider. A Qualified DER can be opted in to be available for energy services by the DER Host.
```

DER Hosts opts DERs into the program after a successful Qualification (Value principle Level 3) by continued staking. A DER that does not continue to fulfill requirements of the Qualification will lose the stake and the ENERGY tokens staked are forfeited and burned, providing an incentive for DER Hosts to maintain DER availability from Qualification and market bidding to the fulfillment of the Client contract.

Clients receive staked, qualified, controllable DERs from the Service Provider and bid these into the market (typically one day in advance). The winning bids generated revenue are allocated from the Service Contract Pool (via HNT DC burn mechanism) to the DER Hosts who have staked controllable DERs (Value principle Level 3).

#### How it works for Battery storage

> Batteries are flexible and can help regulate the grid by both consumption and production of energy, as an example; a battery with 0 % charge can only charge from the grid, and a battery with 100 % can only discharge power to the grid - in order to be flexible both ways the state of charge needs to be in the mid range. Batteries are not affected by weather conditions or time of day and are as such a more valuable resource for the network.

Reviewer: ```suggestion
Batteries are flexible and can help regulate the grid by both consumption and production of energy, as an example; a battery with 0 % charge can only charge from the grid, and a battery with 100 % can only discharge power to the grid - in order to be flexible both ways the state of charge needs to be in the mid-range. Batteries are not affected by weather conditions or time of day and are as such a more valuable resource for the network.
```

Existing Hotspot owners (with a compatible dual mining Hotspot) on the IOT network will be granted one free ENERGY gateway onboard with one free DER onboarding within the first 3 months after dual mining launch.

### Network Rates
At the start, usage of the network is divided into two main buckets, energy data and ancillary services. Usage across all buckets is recorded by network oracles and rewarded to DER Hosts at the end of each epoch. Access to data and DERs for ancillary services is paid using Data Credits derived from HNT. Rewards are allocated to DER Hosts in ENERGY tokens. DER Hosts are eligible for rewards in both buckets simultaneously.

#### Energy Data
Energy data is priced at 1 DC per data point. This data is provided at frequencies of up to one data point per second. Typically, data costs from utility meters (which only measures the amount of energy) are generally billed at about $2.60 USD/month with a transmission rate of one data point per hour. DERs, however, can send more data points more frequently. The ENERGY network prices the data cost for a single DER significantly lower rate than the base price for what a utility company is paying.

> | Frequency    | Data Points (DC) /Day | DC per Data Point | 30 Day Cost (USD) |
> |--------------|-----------------------|-------------------|--------------------|
> | Every Hour   | 24                    | 1                 | $0.0072            |
> | Every Minute | 1&nbsp;440            | 1                 | $0.432              |
> | Every Second | 86&nbsp;400           | 1                 | $25.92              |

Reviewer: ```suggestion
| Frequency    | Data Points (DC) /Day | DC per Data Point | 30 Day Cost (USD) |
|--------------|-----------------------|-------------------|-------------------|
| Every Hour   | 24                    | 1                 | $0.0072           |
| Every Minute | 1&nbsp;440            | 1                 | $0.432            |
| Every Second | 86&nbsp;400           | 1                 | $25.92            |
```

In the years 2023-2024, the average price across several European and US markets ranges from $10-40 USD per hour for a portfolio of 1 MW controllable resources. As a rough example of the historical market, a standard household with a 10 kW battery (a typical battery installation) could range between $2.4 - $9.6 per day ($876 - $3504 per year), equivalent to 240 000 - 960 000 data credits (DC) burn per day. There are already well known services available such as the [Tesla Demand Side Grid Support](https://www.tesla.com/support/energy/virtual-power-plant/dsgs) that compensates battery owners with up to $350 for a year. The ENERGY network rewards will be equivalent or higher depending on region. 

Solar PV systems are only eligible for curtailment during daylight hours with safety margins depending on the weather, making the reward significantly lower than for systems including batteries.

Example of a 24-hour delivery period of aggregated DERs with a combined power of 1 MW (equivalent to 100 average-sized DERs for a household) in a European country with prices as of Q2 2024:

> | Type of DER | Daily availability | Controllable resources in MW | Daily price in $  | Daily price(DC) | Monthly price(USD) |
> |-------------|--------------------|------------------------------|-------------------|-----------------|--------------------|
> | Batteries   | 24&nbsp;h               | 1                            | $1&nbsp;000            | 100&nbsp;000&nbsp;000     | $30&nbsp;000           |
> | Solar PVs   | ~ 8&nbsp;h              | 1                            | $100              | 10&nbsp;000&nbsp;000      | $3&nbsp;000              |

Reviewer: ```suggestion
| Type of DER | Daily availability | Controllable resources in MW | Daily price in $ | Daily price(DC)       | Monthly price(USD) |
|-------------|--------------------|------------------------------|------------------|-----------------------|--------------------|
| Batteries   | 24&nbsp;h          | 1                            | $1&nbsp;000      | 100&nbsp;000&nbsp;000 | $30&nbsp;000       |
| Solar PVs   | ~ 8&nbsp;h         | 1                            | $100             | 10&nbsp;000&nbsp;000  | $3&nbsp;000        |
```

# Rationale
As decentralized physical infrastructure has risen, token incentives are a path forward when it comes to coordinating the deployment of physical resources. Solar and battery storage, as a residential asset class, will benefit from the people-powered growth and management that comes with web3.

Solar PV deployments show no signs of slowing, and the energy they produce can only be managed at the production source. In order to make solar energy work on a global scale, utilities and people need a simple way to communicate and coordinate. Given the funds trading hands at an industrial scale, it stands to reason that an incentive should be passed to the small-scale individuals that make these larger systems work.

## The Power of Decentralized Physical Infrastructure Networks

Decentralized Physical Infrastructure Networks (DePIN) represent a new paradigm in building and managing critical infrastructure. By leveraging blockchain technology and token incentives, DePIN enables the creation of community-owned networks that are more resilient, cost-effective, and adaptable than traditional centralized systems.

### Why the DePIN Approach Makes Sense for Energy

> The energy sector is undergoing a rapid transformation towards decentralization, driven by the proliferation of small-scale renewable energy resources. There needs to be an effective way of coordinating the vast number of small energy resources. By leveraging the community with already open hardware platforms, and coordinating current and unforeseen products, we can build a more cost effective, permissionless, and transparent energy system than options available today. 

Reviewer: ```suggestion
The energy sector is undergoing a rapid transformation towards decentralization, driven by the proliferation of small-scale renewable energy resources. There needs to be an effective way of coordinating the vast number of small energy resources. By leveraging the community with already open hardware platforms, and coordinating current and unforeseen products, we can build a more cost-effective, permissionless, and transparent energy system than options available today. 
```

> The decentralization of the energy sector is a disruptive movement and we need technology that empowers people to be the stakeholders and builders of this movement, not just bystanders.

Reviewer: ```suggestion
The decentralization of the energy sector is a disruptive movement, and we need technology that empowers people to be the stakeholders and builders of this movement, not just bystanders.
```

> The energy system is already a decentralized network that functions solely because there is a coordination of both consumption and production that is always in balance. However it is rapidly growing and needs are outpacing what’s possible with existing tools. We believe that this coordination–built upon a decentralized and transparent protocol where the people participating are the stakeholders–can solve this problem. 

Reviewer: ```suggestion
The energy system is already a decentralized network that functions solely because there is a coordination of both consumption and production that is always in balance. However, it is rapidly growing and needs are outpacing what’s possible with existing tools. We believe that this coordination–built upon a decentralized and transparent protocol where the people participating are the stakeholders–can solve this problem. 
```

One example: In Germany there is now the possibility of installing small scale solar PV that can be plugged directly into your wall socket (such as balcony solar). Bringing solutions like this to scale is a global energy solution, but there are currently no effective incentive mechanisms to get these resources coordinated in the traditional markets for small scale PV or batteries deployment. The ENERGY subnetwork will incentivize and reward individuals for addressing global power needs. 

This shift presents both challenges and opportunities:

- **Coordination of Distributed Energy Resources**: There is a need for an effective way to coordinate the vast number of small energy resources. The DePIN model, with its community-driven approach, offers a solution that is more cost-effective, permissionless, and transparent than traditional centralized systems.
- **Empowering Stakeholders**: The decentralization of the energy sector is a disruptive movement. DePIN technology empowers individuals to be active stakeholders and builders in this movement, rather than passive bystanders.

> - **Balancing Supply and Demand**: While the energy system is inherently decentralized in terms of consumption and now more increasingly reneawable production, its rapid growth is outpacing the capabilities of existing coordination tools. A decentralized and transparent protocol, where participants are stakeholders, can address this coordination challenge.

Reviewer: ```suggestion
- **Balancing Supply and Demand**: While the energy system is inherently decentralized in terms of consumption and now more increasingly renewable production, its rapid growth is outpacing the capabilities of existing coordination tools. A decentralized and transparent protocol, where participants are stakeholders, can address this coordination challenge.
```
